### PR TITLE
Chore/tri 1519 ess top down use case

### PR DESCRIPTION
**Pull request by mkanal**
## Description
- Add activity diagram for ess case 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: docs/src/uml-diagrams/irs-recursive/ess-top-down/1_ess-top-down-sequence-Get_AAS_Assets_by_BPN.puml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    participant decentralDigitalTwinRegistry as dDTR
+end box
+
+
+box SharedServices #DarkGrey
+    participant DiscoveryFinder as DiscoveryFinder
+    participant EDCDiscoveryService as EDCDiscoveryService
+end box
+
+    Client -> IRS : [hidden] GET /irs/proxy/registry/lookup/shells{bpn}
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/irs-recursive/ess-top-down/1_ess-top-down-sequence-Get_AAS_Assets_by_BPN.puml]: Client -> IRS : [hidden] GET /irs/proxy/registry/lookup/shells/{searchBpn}
    note left
        request body
             {providerBpnl="BPNL0123456789XX"};
             {limit=50}
    end note

File: docs/src/uml-diagrams/irs-recursive/ess-top-down/2_ess-top-down-sequence-Get_AAS_by_AasIdentifier_Filter_AsPlanned.puml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    participant DiscoveryFinder as DiscoveryFinder
+    participant EDCDiscoveryService as EDCDiscoveryService
+end box
+
+
+loop array of aasIdentifier
+    Client -> IRS : [hidden] GET /irs/proxy/registry/shell-descriptors/{aasIdentifier}/{bpn}
+    ref over IRS, DiscoveryFinder, EDCDiscoveryService, EDC, dDTR
+         registry/shell-descriptors/{aasIdentifier} for given BPN
+        :return array of {aasIdentifier}
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/irs-recursive/ess-top-down/2_ess-top-down-sequence-Get_AAS_by_AasIdentifier_Filter_AsPlanned.puml]: is fixed loop is closed after request

File: docs/src/uml-diagrams/irs-recursive/ess-top-down/3_ess_top_down_sequence-RegisterEssInvestigationOrder.puml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+end box
+
+Client -> IRS : /ess/bpn/investigation/orders
+note left
+    {
+      "bomLifecycle": "asPlanned","batchSize": 20,
+        "callbackUrl": "https://hostname.com/callback?orderId={orderId}&batchId={batchId}",
+      ...
+      "incidentBPNSs": [
+        "BPNL666666666666","BPNL666666666667"
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/irs-recursive/ess-top-down/3_ess_top_down_sequence-RegisterEssInvestigationOrder.puml]: resolved switch to BPNS

File: docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence.puml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+end note
+IRS -> IRS : register an IRS ess incident job
+
+ref over IRS, DiscoveryFinder, EDCDiscoveryService, EDC, dDTR
+    lookup/shells for given globalAssetId
+    :return AAS for globalAssetId
+end
+IRS -> SubmodelServer : GET SingleLevelUsageAsPlanned
+IRS <-- SubmodelServer : Receiver SingleLevelBomAsPlanned
+IRS --> IRS : Extract childCXIds from SingleLevelBomAsPlanned
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence.puml]: resolved

File: docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence-highlevel.puml [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+IRS -> IRS : register an IRS ess incident job
+
+ref over IRS, DiscoveryFinder, EDCDiscoveryService, EDC, dDTR
+    lookup/shells for given globalAssetId
+    :return aasIdentifier
+    /registry/shell-descriptors/{aasIdentifier}
+    :return AAS for aasIdentifier
+end
+IRS -> SubmodelServer : GET SingleLevelUsageAsPlanned
+IRS <-- SubmodelServer : Receiver SingleLevelBomAsPlanned
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence-highlevel.puml]: resolved

File: docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence-highlevel.puml [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    Client -> IRS : [hidden] GET /irs/proxy/registry/lookup/shells{bpn} {bpnl="BPNL0123456789XX"}
+
+    ref over IRS, DiscoveryFinder, EDCDiscoveryService, EDC, dDTR
+        lookup/shells for given BPN
+        :return array of {aasIdentifier}
+    end
+    Client <-- IRS :return array of aasIdentifier
+
+
+loop array of aasIdentifier
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence-highlevel.puml]: fixed

File: docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence-highlevel.puml [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+      "incidentBpns": [
+        "string"
+      ],
+      "key": {
+        "bpn": "BPNL0123456789XX",
+        "globalAssetId": "urn:uuid:6c311d29-5753-46d4-b32c-19b918ea93b0"
+      }
+    }
+end note
+IRS -> IRS : register an IRS ess incident job
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/irs-recursive/ess-top-down/ess-top-down-sequence-highlevel.puml]: not with current modifications to asPlanned

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**